src/ui/popup/components/site-toggle/index.tsx

```diff
@@ -1,14 +1,23 @@
 import {m} from 'malevic';
 import CheckmarkIcon from './checkmark-icon';
 import {Button} from '../../../controls';
-import {getURLHost, isURLEnabled} from '../../../../utils/url';
+import {getURLHost, isURLEnabled, isPDF} from '../../../../utils/url';
 import {ExtWrapper, TabInfo} from '../../../../definitions';
 
 export default function SiteToggleButton({data, tab, actions}: ExtWrapper & {tab: TabInfo}) {
+
+    function click() {
```

Comment:
There is a convention :) ... to start event handlers with `on` (like `onClick`), to describe what is being clicked (like `onSiteToggleClick`) or describe the event, rather than DOM action (`onSiteToggle`, maybe it could be reaction on click, keypress etc). Some people prefer to add `-handler` or `-listener` (like `siteToggleHandler`).